cmd/cli/config/list.go

```diff
@@ -40,6 +41,7 @@ Each key shown can be used with:
 			if err != nil {
 				return err
 			}
+			log.Info().Msgf("Config loaded from: %s, and with data-dir %s", cfg.Paths(), cfg.Get(types.DataDirKey))
```

Comment:
I understand your point, though we should still log that configs are NOT loaded from a config file. Instead of having different log lines that users grep for, it might be better just having a single type of log